content/020_prerequisites/automatic_cloud9/workspace.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
I'd rather see this tabularized under one set of instructions for creating the cloud9 instance (rather than two different entries/chapters on the left index.)

Reply:
not sure what you mean. maybe you can clarify :)